src/discord.cpp

```diff
@@ -20,30 +43,60 @@ Discord::Discord(QObject *parent) : QObject(parent)
 
     Discord_Initialize(APPLICATION_ID, &handlers, 1, "1234");
 
-    // start Discord's event loop at 50ms
+    // process Discord callbacks every 50ms
     startTimer(50);
 }
 
 
 Discord::~Discord()
 {
-    Discord_Shutdown();
+    //    Discord_Shutdown();
+}
+
+void Discord::setGameName(const QString& name)
+{
+    if (!mLoaded) {
+        return;
+    }
+
+    char buffer[256];
+    sprintf(buffer, "Playing %s", name.toUtf8().constData());
```

Comment:
Ehh... I am not sure this is a good idea. If it shows for an italian user "giocando a clessindra", nobody elsewhere will know what are they doing.

Reply:
Consider the Spanish players, previously using MudletRL, are they going to want to see:

    Jugando: Reinos de leyenda

or:

    Playing: Reinos de leyenda

Reply:
I was looking to see how well developed Discord was for non-Americanish languages to see whether it is likely they'd look at this but was not able to find anything out. Also, it is not something that **[this](http://nvu.io/bots/discord-translator/)** bot is going to solve - and since they dropped their free service on 2018/05/17 it is not one that we can afford to invite to our channel... 🙁

Reply:
Discord does this just fine. Using Discord with German client, I would see "Spielt Mudlet" and not "Playing Mudlet". Here is a screenshot for future reference: 
![image](https://user-images.githubusercontent.com/117238/40687162-08764c66-639a-11e8-9fef-cb2a036f3573.png)